code/src/css/MoviesDetails.css

```diff
@@ -0,0 +1,110 @@
+ .movie-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    width: 100%;
+    height: 100%;
+ }
+ 
+ .background {
+    position: relative ;
+    min-height: 150vh;
+    object-fit: cover;
+}
+
+.summary {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    padding: 50px;
+
+}
+
+.details {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    background: rgba(0, 0, 0, 0.5);
+    padding: 20px;
+}
+
+.back-button {
+    border-radius: 50px;
+    border: none;
+    width: 20vw;
+    height: 4vh;
```

Comment:
Interesting to see these units being used, and to see a button be responsive in size!